src/content/docs/dropins/cart/cart-functions.mdx

```diff
@@ -298,118 +394,147 @@ type StoreConfigModel = {
-  total: {
-    includingTax: Price;
-    excludingTax: Price;
-  };
-  discount?: Price;
-  subtotal: {
-    excludingTax: Price;
-    includingTax: Price;
-    includingDiscountOnly: Price;
-  };
-  appliedTaxes: TotalPriceModifier[];
-  totalTax?: Price;
-  appliedDiscounts: TotalPriceModifier[];
-  shipping?: Price;
-  isVirtual?: boolean;
-  addresses: {
-    shipping?: {
-      countryCode: string;
-      zipCode?: string;
-      regionCode?: string;
-    }[];
-  };
-  isGuestCart?: boolean;
-};
+export const refreshCart = async (): Promise<CartModel | null>
+```
+
+### Returns
+
+Returns a promise that resolves to a `CartModel` object or null. The `CartModel` object has the following shape:
+
+<CodeInclude code={cartModel} title="CartModel object" lang="ts" frame="none" />
+
+### Usage
+
+To refresh the cart data:
+
+```ts
+import { refreshCart } from '@/cart/api/refreshCart';
+
+refreshCart();
 ```
 
 ## resetCart
 
-A function that resets the cart.
+This function resets the cart. As a result, the cart ID is set to null and the authenticated status is set to false.
+
+```ts
+export const resetCart = (): Promise<CartModel | null>
+```
+
+### Returns
+
+Returns a promise that resolves to a `CartModel` object or null. The `CartModel` object has the following shape:
+
+<CodeInclude code={cartModel} title="CartModel object" lang="ts" frame="none" />
+
+### Usage
+
+To reset the cart:
 
 ```js
 import { resetCart } from '@/cart/api/resetCart';
 
 resetCart();
 ```
 
-Here's the shape of the data returned from the `resetCart` function:
+## updateProductsFromCart
 
-```ts
-type Cart = {
-  cartId: null;
-  authenticated: false;
-};
+The `updateProductsFromCart` function updates the quantity of a product in the cart. The specified quantity replaces the current quantity. Setting the quantity to 0 removes an item from the cart. The function calls the [`updateCartItems`](https://developer.adobe.com/commerce/webapi/graphql/schema/cart/mutations/update-items/) mutation.
 
+```ts
+export const updateProductsFromCart = async (
+  items: UpdateProductsFromCart
+): Promise<CartModel | null>
 ```
 
-## updateProductsFromCart
+<OptionsTable
+  compact
+  options={[
+    ['Parameter', 'Type', 'Req?', 'Description'],
+    ['items','UpdateProductsFromCart','Yes','An input object that defines products to be updated.'],
+  ]}
+/>
+
+The `UpdateProductsFromCart` object has the following parameters:
+
+<OptionsTable
+  compact
+  options={[
+    ['Parameter', 'Type', 'Req?', 'Description'],
+    ['uid', 'string', 'Yes', 'The unique ID of an item in the cart.'],
+    ['quantity', 'number', 'Yes', 'The new quantity of the item.'],
+  ]}
+/>
+
+### Returns
+
+Returns a promise that resolves to a `CartModel` object or null. The `CartModel` object has the following shape:
 
-A function that updates products in cart. It takes the `uid` and `quantity` parameters.
+<CodeInclude code={cartModel} title="CartModel object" lang="ts" frame="none" />
+
```

Comment:
**Suggestion:** This function emits the `cart/updated` and `cart/data` events as well as publishing add to cart or remove from cart events to the Adobe Commerce Data Layer (ACDL).